### PR TITLE
Include codecs in canPlayType check

### PR DESCRIPTION
**Pull request by robwalch**
### This PR will...

Include `source.mediaTypes` codecs when present in `canPlayType` check.
 
### Why is this Pull Request needed?

So that the html5 provider does not attempt to handle DASH streams in Edge when the only video codec used is VP9.

### Are there any Pull Requests open in other repos which need to be merged with this?

https://github.com/jwplayer/jwplayer-commercial/pull/5778

#### Addresses Issue(s):

JW8-1405

### CODE REVIEW

File: test/unit/api-get-config-test.js [hunk 1]
```diff
@@ -5,11 +5,13 @@ import sinon from 'sinon';
...
 describe('Api.getConfig', function() {
 
+    const sandbox = sinon.sandbox.create();
+
     beforeEach(() => {
         const container = document.createElement('div');
         container.id = 'player';
         document.body.appendChild(container);
-        console.error = sinon.stub();
+        sandbox.spy(console, 'error');
```
Comment (egreaves) [inline test/unit/api-get-config-test.js]: I like the use of `sandbox`

File: test/unit/api-get-config-test.js [hunk 2]
```diff
@@ -21,7 +23,7 @@ describe('Api.getConfig', function() {
         for (let i = instances.length; i--;) {
```
Comment (johnBartos) [inline test/unit/api-get-config-test.js]: `sandbox.restore()` in afterEach is a good idea

File: test/unit/api-test.js [hunk 3]
```diff
@@ -39,8 +42,9 @@ describe('Api', function() {
         for (let i = instances.length; i--;) {
             instances[i].remove();
         }
-        utils.log.reset();
-        console.error.reset();
+        utils.log.restore();
```
Comment (johnBartos) [inline test/unit/api-test.js]: `sandbox.restore` hits all sinon constructs

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🔎 Assign some reviewers or assignees.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**